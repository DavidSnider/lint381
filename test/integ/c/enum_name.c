enum foo {};

enum Foo_e {};
