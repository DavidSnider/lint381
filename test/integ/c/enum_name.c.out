enum_name.c:1:6: error: Enum name 'foo' should be capitalized
enum_name.c:1:6: error: Enum 'foo' should end with '_e'
